.github/merge_rules.json

```diff
@@ -99,12 +99,12 @@
       "patterns": [
          "aten/src/ATen/native/cuda/linalg/**",
          "aten/src/ATen/LinalgBackend.h",
-         "aten/src/ATen/native/**/*LinearAlgebra*",
+         "aten/src/ATen/native/**LinearAlgebra*",
```

Comment:
Can you please add a unit test to make sure it would include both aten/src/ATen/native/LinearAlgebra.cpp as well as aten/src/ATen/native/cpu/LinearAlgebraKernel.cpp?